daffodil-runtime1/src/main/scala/org/apache/daffodil/processors/DaffodilParseXMLReader.scala

```diff
@@ -144,7 +142,13 @@ class DaffodilParseXMLReader(dp: DataProcessor) extends DFDL.DaffodilParseXMLRea
   }
 
   def parse(isdis: InputSourceDataInputStream): Unit = {
-    val sio = createSAXInfosetOutputter(this)
+    val sio = createSAXInfosetOutputter(this,
+      saxNamespaceFeatureValue,
+      saxNamespacePrefixesFeatureValue)
+    // validate that the features are not false/false
+    if (!sio.namespacesFeature && !sio.namespacePrefixesFeature) {
+      throw new SAXException("Illegal State: Features cannot both be false")
```

Comment:
Might be worth mentioning specifically which features, that might not always be obvious to people not faimilar with SAX. Probably not too confusing right now since Daffodil only supports two features, but if ever add more in the future it could be helpful.